Source/Core/EllipsoidTangentPlane.js

```diff
@@ -117,7 +132,7 @@ define([
     var projectPointOntoPlaneCartesian3 = new Cartesian3();
 
     /**
-     * Computes the projection of the provided 3D position onto the 2D plane.
+     * Computes the projection of the provided 3D position onto the 2D plane, radially outward from the global origin.
```

Comment:
Probably "WGS84 coordinate system origin" here and below.  WGS84 is our global coordinate system, but "global" is probably not explicit enough here.